sympy/solvers/ode.py

```diff
@@ -6468,7 +6467,7 @@ def _linear_2eq_order1_type1(x, y, t, r):
 
     """
     l = Dummy('l')
-    C1, C2, C3, C4 = symbols('C1:5')
+    C1, C2, C3, C4 = get_numbered_constants((x(t), y(t)), num=4)
```

Comment:
hmmm, but if the DE if like `Eq(diff(x(t),t), C1*x(t)+y(t))...` then this wouldn't work?

instead of `x(t), y(t)` we need the thing called `eq` in `sysode_linear_2eq_order1`.  But we don't have that here...  Would `get_numbered_constants((r, x(t), y(t)), num=4)` work?  Or maybe you have another workaround?